src/chains/migrations/0028_chain_vpc_transaction_service_uri.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.7 on 2021-09-27 09:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chains", "0027_chain_description"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="chain",
+            name="vpc_transaction_service_uri",
+            field=models.URLField(default=""),
+            preserve_default=False,
```

Comment:
Should we constrain this to be non-blank and non-nullable?

Reply:
`non-blank` and `non-nullable` are the defaults: `blank=False, null=False` for all Django Fields.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#null
https://docs.djangoproject.com/en/3.2/ref/models/fields/#blank